testcharms/charm-repo/bionic/dummy-no-series/metadata.yaml

```diff
@@ -3,4 +3,3 @@ summary: "That's a dummy charm."
 description: |
     This is a longer description which
     potentially contains multiple lines.
-minjujuversion: "1.0.0"
```

Comment:
Does this charm still deploy if it's got no series and has no manifest?

Reply:
This charm was intended to fail to deploy, since a charm with no series in metadata or bases in manifest is invalid

However, we needed to add an empty manifest, otherwise we would recieve the wrong error